pkg/resource/podtemplatespec.go

```diff
@@ -229,7 +229,24 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, proxyLister configl
 	}
 	volumes = append(volumes, vol)
 	mounts = append(mounts, corev1.VolumeMount{Name: vol.Name, MountPath: "/etc/pki/ca-trust/source/anchors"})
-	deps.AddConfigMap(vol.VolumeSource.ConfigMap.LocalObjectReference.Name)
+	deps.AddConfigMap(v1.ImageRegistryCertificatesName)
```

Comment:
I guess I crossed the boundaries here, I have also implemented this on https://github.com/openshift/cluster-image-registry-operator/pull/342